### ISSUE TITLE
Bug: Empty Index Panel Buttons have 4px borderRadius instead of 6px

### ISSUE THREAD

**Issue by Jesse-Box** [getsentry/sentry#66979]
4px
![Image](https://github.com/getsentry/sentry/assets/6508377/2bac41b1-0f0e-4e95-9b44-3ed2765818d4)

default (6px)
![Image](https://github.com/getsentry/sentry/assets/6508377/2c32ada7-cf52-4fe5-9460-73fda278f6a8)

Not exclusive to Replay either, you can see it on the crons empty state

**Comment by evanpurkhiser** [comment]
@vuluongj20 is this related to the border context

**Comment by vuluongj20** [comment]
@evanpurkhiser Yes. The context is useful in most cases but then there are edge cases like this. We can add a prop to disable [`PanelProvider`](https://github.com/getsentry/sentry/blob/master/static/app/utils/panelProvider.tsx#L23)'s effect.
